sdk/storage/Azure.Storage.Files.DataLake/src/DataLakePathClient.cs

```diff
@@ -1213,7 +1239,9 @@ internal virtual void SetNameFieldsIfNull()
         /// a failure occurs.
         /// </remarks>
         private async Task<Response<PathInfo>> SetAccessControlInternal(
```

Comment:
According to the comment in https://apiview.azurewebsites.net/Assemblies/Review/548418d761fb45ce9fc98c8cfa9f5ef8/e248475ab79e41b7b30e946ffa26553e
it says the return type should be PathAccessControl.